### ISSUE TITLE
FR: Dynamic theme switching in menu bar

### ISSUE THREAD

**Issue by pat-s** [go-gitea/gitea#14408]
I would like to switch between the dark/light theme more often during the day.

It would be awesome if the switch would be accessible more easily, e.g. in the menu bar.
Currently it is within the account settings and requires to hit a "save settings" button explicitly which is quite a long journey for a repetitive task.

Would be awesome if this feature would land! 🚀

**Comment by silverwind** [comment]
Yes, a theme switcher dropdown in the footer would be nice. Ideally without triggering a page reload.

**Comment by a1012112796** [comment]
Maybe https://github.com/go-gitea/gitea/pull/11807 is what you want...

**Comment by pat-s** [comment]
Looks promising. Is the PR stalled or are you planning to continue working on it?

**Comment by pat-s** [comment]
Related to #10927

**Comment by wxiaoguang** [comment]
Done
